app/src/org/commcare/fragments/personalId/PersonalIdBackupCodeFragment.java

```diff
@@ -258,6 +258,7 @@ private void handleFailedBackupCodeAttempt() {
 
     private void handleAccountLockout() {
         logRecoveryResult(false);
+        FirebaseAnalyticsUtil.reportPersonalIdConfigurationFailure(AnalyticsParamValue.START_CONFIGURATION_LOCKED_ACCOUNT_FAILURE);
```

Comment:
@OrangeAndGreen I see this is also being logged by `handleCommonSignupFailures` ? and I am confused what path is the right one that gets triggered on locked account ?